Dockerfile

```diff
@@ -71,13 +71,17 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
 	adduser user sudo && \
 	echo "%users ALL=(ALL:ALL) NOPASSWD: ALL" | sudo tee /etc/sudoers.d/users
 
-# Install setuptools, wheel and pip for Python3
+# Install pip for Python3
 RUN wget https://bootstrap.pypa.io/get-pip.py -O - | python3
+# Install virtualenv and httpie
+RUN pip install virtualenv httpie
```

Comment:
Curious why virtualenv instead of python3's venv?

Reply:
@miigotu excellent question and I was asking myself that exact question and had intended to revisit this.

After testing I discovered that as a python2 compatible pip is being installed after latest greatest pip for python3, it happens that default `pip` ends-up pointing to the older version used for python2.

So I now adjusted the code so that it calls `pip2` or `pip3` always to enforce the backend python version.  Now, as I ain't sure if `httpie` is only needed against python2 or not (and why I don't recall) it now install it for both versions of python.